quizapp/quiz-loader/src/main/java/billennium/DataLoader.java

```diff
@@ -0,0 +1,84 @@
+package billennium;
+
+import billennium.quizapp.entity.Answer;
+import billennium.quizapp.entity.Question;
+import billennium.quizapp.entity.QuizDefinition;
+import billennium.quizapp.repository.QuizDefinitionRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.boot.ApplicationArguments;
+import org.springframework.boot.ApplicationRunner;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+@Service
+@RequiredArgsConstructor
+public class DataLoader implements ApplicationRunner {
+
+    private final QuizDefinitionRepository quizDefinitionRepository;
+
+    @Override
+    public void run(ApplicationArguments args) {
+        QuizDefinition quizDefinition = QuizDefinition.builder()
+                .title("JAVA")
+                .questions(createListQuestions())
+                .build();
+        quizDefinitionRepository.save(quizDefinition);
+    }
+
+    private List<Question> createListQuestions() {
+        Question question1 = Question.builder()
+                .text("Java to:")
+                .answers(createListAnswersForQuestion("Język programowania", "JVM", "C++"))
+                .build();
+        Question question2 = Question.builder()
+                .text("JVM TO")
+                .answers(createListAnswersForQuestion("JRE", "JDK", "JAVA VIRTUAL MACHINE"))
+                .build();
+        Question question3 = Question.builder()
+                .text("Co to jest konstruktor domyślny?")
+                .answers(createListAnswersForQuestion("Jest tworzony jeśli nie zdefiniowano jawnie konstruktora w klasie",
+                        "Jest tworzony tylko wtedy kiedy zostanie jawnie zdefiniowany w kodzie",
+                        "Konstruktor zdefiniowany ze słowem kluczowym default"))
+                .build();
+        Question question4 = Question.builder()
+                .text("Czy można zmienić istniejący String?")
+                .answers(createListAnswersForQuestion("Tylko za pomocą StringBuilder-a", "NIE", "Tak," +
+                        " jak każdy inny obiekt"))
+                .build();
+        Question question5 = Question.builder()
+                .text("Czy klasa może dziedziczyć kilka interfejsów?")
+                .answers(createListAnswersForQuestion("Tylko wtedy jak interfejsy roszerzają jeden wspólny interfejs",
+                        "NIE", "TAK"))
+                .build();
+
+        List<Question> listQuestion = Stream.of(question1, question2, question3, question4, question5).collect(Collectors.toList());
```

Comment:
to dziwne - robisz strumień by z niego następnie zrobić liste. Tutaj użyłbym `Arrays.asList()`. Nawet bym nie przypisywał obiektu tylko od razu zwracał listę z metody.

Reply:
tutaj IDE pewnie nawet podpowiada że przypisanie jest zbędne